install/kubernetes/cilium/templates/cilium-operator-clusterrole.yaml

```diff
@@ -41,14 +41,16 @@ rules:
 {{- end }}
 {{- end }}
 {{- end }}
-{{- if or .Values.operator.removeNodeTaints .Values.operator.setNodeNetworkStatus }}
+{{- if or .Values.operator.removeNodeTaints .Values.operator.setNodeNetworkStatus .Values.operator.endpointGCInterval }}
 - apiGroups:
   - ""
   resources:
   - nodes
   verbs:
   - list
   - watch
+{{- end }}
+{{- if or .Values.operator.removeNodeTaints .Values.operator.setNodeNetworkStatus }}
 - apiGroups:
   - ""
   resources:
```

Comment:
@sayboras missing `delete` on `ciliumnodes` from [this upstream commit](https://github.com/cilium/cilium/pull/19576/commits/f4aa177df423a6469bd0deb9cec8d6b2bc5912f6) in this file, because in v1.10 we have `*` on `ciliumnodes`, se bellow.